### ISSUE TITLE
Gather Voting System Requirements for the Oct. 2018 Member Meeting

### ISSUE THREAD

**Issue by Jake-Gillberg** [rchain/bounties#399]

**Comment by dckc** [comment]
@Jake-Gillberg tell me some stories?
  - a story about how things should work
  - a story about a bad thing that could / would happen if we get it wrong.

I have good experience with [StoryTellingAndTestCases](https://informatics.gpcnetwork.org/trac/Project/wiki/StoryTellingAndTestCases) for software development.

**Comment by dckc** [comment]
See also #209 ; are they both worth tracking? Or is one a dup?

**Comment by Jake-Gillberg** [comment]
@dckc I view #209 as a request for voting tools with a wider and more generic scope.

**Comment by dckc** [comment]
@Jake-Gillberg I guess you should know:
> RESOLVED: Jake has authority to draft a Voting application and ask for the needed resources to implement by October members meeting -- [executive committee March 2](https://docs.google.com/document/d/1MeYZFva1U-5Faf7NZmOrd0KeJ1CAc293sExC7Lxa4Ec/edit#)

cc @kennyrowe

**Comment by barneycinnamon** [comment]
@Jake-Gillberg Have you been working on this without adding comments here that reflect work that has been completed to date? I want to make sure you are getting the support you need to tackle this important issue.

**Comment by dckc** [comment]
I recently stumbled across https://github.com/rchain/all-members-meeting ; I wonder how it relates.

**Comment by barneycinnamon** [comment]
@Jake-Gillberg How is this going? Have you resolved most or all of the objectives? @deannald @evanjensen

**Comment by deannald** [comment]
@barneycinnamon Yes, requirements gathering for this year's solution is complete.
